core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -371,6 +388,10 @@ public void onLoadChildren(@NonNull String parentId,
         if (parentId.equals(getResources().getString(R.string.app_name))) {
             // Root List
             mediaItems.add(createBrowsableMediaItemForRoot());
```

Comment:
Can you go a bit more into detail why you think that this resolves the issue?
To be honest, I'm not even sure what exactly Google is complaining about. I don't really see any issue... (Testing with the Desktop Head Unit)